src/web/components/MostViewed/MostViewedFooter/MostViewedFooterData.tsx

```diff
@@ -34,7 +36,11 @@ const secondTierStyles = css`
 	}
 `;
 
-function buildSectionUrl(ajaxUrl: string, sectionName?: string) {
+function buildSectionUrl(
+	ajaxUrl: string,
+	pillar: CAPIPillar,
```

Comment:
Are we using this in this function? I presume we _will_ be but I'm surprised eslint allows it to be there if it's not used now